aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh

```diff
@@ -10,7 +10,7 @@ if [ "$VIRTUAL_ENV" == "" ]; then
     (cd "$sourceDir" && pip install .)
 fi
 
-pip install requests lambda-packages credstash
+pip install requests lambda-packages cryptography==2.7 credstash==1.16.1
```

Comment:
credstash needs dependency: cryptography>=2.1. If credstash's version is 1.16.2, cryptography version 2.8 will be installed, but this cryptography version will caused two ImportErrors:
    - No module named asn1crypto.keys,
    - Cannot import name certificate_transparency.

Even if we downgrade credstash version to 1.16.1 or to 1.15.0, pip will still keep cryptography v2.8 as it satisfies the condition <= 2.1. So, explicitly install cryptography v2.7 will fix the errors.

Reply:
Sounds good.